tmt/plans/integration.fmf

```diff
@@ -11,7 +11,7 @@ prepare:
   # reboot in ansible is the only way to reboot in tmt prepare
   - how: ansible
     playbook:
-      - https://github.com/henrywang/bootc/raw/refs/heads/gating/hack/packit-reboot.yml
+      - https://github.com/bootc-dev/bootc/raw/refs/heads/main/hack/packit-reboot.yml
```

Comment:
Wait but can't we just specify running the playbook from the local git checkout?

Reply:
And I'm working on another PR to use `.src.rpm` instead of local git checkout. Just for working with dist-git gating test.